### DISCUSSION TITLE
How to make this homepage effect?

### DISCUSSION THREAD

**Discussion by vera1024** [xu8117/xu8117#1] [Q&A] [answered]
How to make this homepage effect like you?

**Comment by xu8117** [comment, marked as answer]
This is a small and easy trick, for beautify your personal GitHub page.
👨‍💻🎨

1️⃣
Oper your GitHub page, create a new repository, and give it the name of your GitHub profile.
If it's all good, you can see a message like this:
![image](https://user-images.githubusercontent.com/39123862/178944871-39465db9-891b-4464-aa9d-3ff0acc69d51.png)

Alt Text

2️⃣
Create a README.md file inside your new repository, give it a style (you can fine a lot of example online).
![image](https://user-images.githubusercontent.com/39123862/178944918-594cdc5c-dbc1-4873-bf6b-475449dfd932.png)

Alt Text

3️⃣
Commit you README and "et voilà", that's all folks: now you can find a new section on the right of your profile picture.
![image](https://user-images.githubusercontent.com/39123862/178944940-86da80bf-9049-4978-bdb4-a7e4efdb33af.png)

Alt Text

Hope I help you; it's an easy way with a good result! 🖼
